plugins/ptp_operator/metrics/metrics.go

```diff
@@ -210,7 +212,10 @@ func (p *PTPEventManager) ExtractMetrics(msg string) {
 		switch interfaceName {
 		case ClockRealTime: // CLOCK_REALTIME is active slave interface
 			// copy  ClockRealTime value to current slave interface
-			if r, ok := ptpStats[master]; ok && r.Role() == types.SLAVE { // publish event only if the master role is active
+			if masterOffsetSource == ts2phcProcessName {
+				//TODO: once ts2phc events are identified we need to add that here
+				p.GenPTPEvent(profileName, ptpStats[interfaceType], interfaceName, int64(ptpOffset), syncState, eventType)
```

Comment:
will this be removed when we finish the above TODO or do we send both events in the `ts2phc` case

Reply:
Yes, we are not handling GM events now and we don't know what the GM events are going to look like.